.eslintrc.json

```diff
@@ -0,0 +1,20 @@
+{
+  "env": {
+    "node": true,
+    "es2020": true
+  },
+  "extends": ["airbnb-base",  "plugin:prettier/recommended"],
+  "plugins": ["prettier"],
+  "parserOptions": {
+    "ecmaVersion": 12,
+    "sourceType": "module"
+  },
+  "rules": {
+    "prettier/prettier": "error",
```

Comment:
So this `"prettier/prettier": "error"` rule is not needed anymore, as there is prettier plugin extended

Reply:
Removed